.travis.yml

```diff
@@ -71,19 +71,25 @@ install:
   - phpenv config-add drupal.php.ini
   - phpenv rehash
 
-  # Build the Lightning code base.
+  # We need Phing to execute our build steps.
   - composer global require phing/phing
-  - phing build -Dversion=$VERSION -Dbranch=$TRAVIS_COMMIT
-
-  # Install Lightning.
-  - phing install -Ddb.database=drupal -Dversion=$VERSION
-  # Execute database updates if we're not building from HEAD.
-  - if [ $VERSION != "HEAD" ]; then phing update; fi
+  # Build the Lightning code base.
+  - composer install
+  # Install Lightning cleanly so that settings.php will be created properly.
```

Comment:
There's [this: Settings.php generator](https://www.drupal.org/project/settingsphp), but I think our time is better spent other places than getting that stable :)